benches/bin/bench-tikv.rs

```diff
@@ -96,6 +98,37 @@ fn main() {
         );
     }
     // TODO allow user to specify flag to just bench some cases.
-    raftstore::bench_raftstore();
-    mvcc::bench_engine();
+
+
+    let mut args: Vec<_> = env::args().skip(1).collect();
+
+    let available_options = vec!["raftstore", "tombstone-scan"];
```

Comment:
why rename to "tombstone-scan"? maybe "mvcc" here?

Reply:
It does bench "tombstone-scan". In `benches/bin/mvcc.rs`. I don't know why either.

Reply:
this is still included in the mvcc.

Here I think you should use mvcc.